06-Tables/Advanced_Tables.ipynb

```diff
@@ -1172,9 +1172,9 @@
     }
    },
    "source": [
-    "### Quantity: doesn't Table already support units?\n",
+    "### Quantity: doesn't `Table` already support units?\n",
     "\n",
-    "We saw in the `sources` table that we can define unit.  Aren't we good to go?  **No**"
+    "We saw in the `sources` table that we can define unit.  Aren't we good to go?  **Not quite!**"
```

Comment:
Good catch at the end of this line. I'm wondering if earlier should be "can define units"? Or maybe "can define a unit"?

Reply:
Good eye, I missed this! I'll change it to "units."